docs/detailed_examples.rst

```diff
@@ -50,9 +50,8 @@ downloaded from the examples_ section of the repository.
 
 **Environmental data**
 
-This dataset has been used in a benchmarking exercise, see https://github.com/ec-benchmark-organizers/ec-benchmark .
-The dataset was derived from NDBC buoy 44007, see https://www.ndbc.noaa.gov/station_page.php?station=44007 . The
-datasets are available here: data_.
+This dataset has been used in a benchmarking exercise, see benchmark_ .
```

Comment:
I don't have a strong opinion on it, but I think I prefer it if we write out external links (the original version).